### PR TITLE
Create model

### PR DESCRIPTION
**Pull request by Honcha-renko**
create model

### CODE REVIEW

File: app/models/task.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 class Task < ApplicationRecord
+  enum tag: %i[postponed important urgent]
```
Comment (aantsyferova) [inline app/models/task.rb:5]: Please, enable rubocop and follow its offences.

File: app/models/task.rb [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+
+  validates :title, presence: true, length: { maximum: 100 }
+  validates :done, presence: true
```
Comment (aantsyferova) [inline app/models/task.rb]: Useless validation for field that has `default: false`

File: db/migrate/20200430223132_create_tasks.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  def change
+    create_table :tasks do |t|
+      t.string :title, null: false
+      t.text :text
+      t.integer :tag
+      t.boolean :done, default: false
+      t.datetime :expire_on
+
+      t.timestamps
+
```
Comment (aantsyferova) [inline db/migrate/20200430223132_create_tasks.rb]: Remove empty lines

File: app/models/task.rb (hunk 1 above)
Comment (Honcha-renko) [reply app/models/task.rb:5]: Rubocop says nothig about this row.
But i added some comments as it recommends.

Comment (aantsyferova) [reply app/models/task.rb:5]: And what about https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment ?

**Pull request merged**